### PR TITLE
test email with oda_api

### PR DESCRIPTION
**Pull request by burnout87**
probably not the cleanest way, but I think it shows that emails are sent when they should and when they should not

### CODE REVIEW

**Comment by burnout87** [comment]
I also applied a change to how the status of a job was applied, but I am not sure if it makes sense of the logic behind the first time this was implemented

File: cdci_data_analysis/flask_app/dispatcher_query.py [hunk 1]
```diff
@@ -209,6 +209,7 @@ def __init__(self, app, instrument_name=None, par_dic=None, config=None, data_se
             query_out.set_query_exception(
                 e, 'InstrumentQueryBackEnd constructor', extra_message='InstrumentQueryBackEnd constructor failed')
 
+            #TODO I guess these comments are no longer needed
```
Comment (volodymyrss) [inline cdci_data_analysis/flask_app/dispatcher_query.py:212]: indeed

**Review by volodymyrss — APPROVED** [review]

File: cdci_data_analysis/analysis/queries.py [hunk 2]
```diff
@@ -500,6 +500,8 @@ def get_query_products(self,instrument,job,run_asynch,query_type='Real',logger=N
 
                 if job_status=='done':
                     job.set_done()
+                elif job_status == 'failed':
+                    job.set_failed()
```
Comment (burnout87) [inline cdci_data_analysis/analysis/queries.py:504]: forgot to ask, is this ok to be done here ?

Comment (volodymyrss) [reply cdci_data_analysis/analysis/queries.py:504]: I guess so.

**Pull request merged**